src/www/interfaces.php

```diff
@@ -359,6 +359,7 @@ function get_wireless_channel_info($interface) {
     $pconfig['lock'] = isset($a_interfaces[$if]['lock']);
     $pconfig['blockpriv'] = isset($a_interfaces[$if]['blockpriv']);
     $pconfig['blockbogons'] = isset($a_interfaces[$if]['blockbogons']);
+    $pconfig['ignore_hotplug_events'] = isset($a_interfaces[$if]['ignore_hotplug_events']);
```

Comment:
same here

Reply:
The lines around this for the existing checkbox options use isset - happy to change but does that mean the existing code is wrong?

Reply:
@nallar I see, you're right, it writes a new array structure instead of using the existing. Better to keep it aligned with the rest, although I would replace the first occurrence in rc.linkup, when set with 0 we should also consider the item false.

Reply:
Ok, will do.